includes/classes/productPulldown.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+    class productPulldown extends pulldown
+    {
+        private $allowed_sort_array = [
+            'products_name',
+            'products_model',
+            'products_name',
+            'products_id',
+            'products_price',
+            'products_sort_order',
+        ];
+        private $db_table_prefix_array = ['pd.', 'p.', 'p.', 'p.', 'p.', 'p.'];
```

Comment:
I'm not sure I like the fragility of splitting these off into a separate property like this.

Perhaps `$allowed_sort_array` could include both?
eg: `'products_name' => 'pd.',`
and then do lookups using isset() or array_key_exists or similar?

Reply:
i'll take a look.  it makes sense to me.